dgram.js

```diff
@@ -849,12 +855,12 @@ export class Socket extends EventEmitter {
    */
   send (buffer, ...args) {
     const id = this.id || rand64()
-    let offset = 0, length, port, address, cb = defaultCallback(this)
+    let offset = 0; let length; let port; let address; let cb = defaultCallback(this)
```

Comment:
lol we should probably just new line these 😅

Reply:
oops, that was `npx standerd --fix .`